src/Avalonia.Diagnostics/Diagnostics/ViewModels/MainViewModel.cs

```diff
@@ -41,6 +41,12 @@ public MainViewModel(TopLevel root)
             Console = new ConsoleViewModel(UpdateConsoleContext);
         }
 
+        public bool FreezePopups
```

Comment:
I'm wondering if we shouldn't show something in status bar (like "Popups frozen") since right now it is hard to tell when it is enabled.

Reply:
![image](https://user-images.githubusercontent.com/4670166/128191108-dffd467a-8877-439c-b05a-2f49464fe971.png)
 Added a small label. Doesn't look that great, but it gets the job done for now :)